CHANGES.rst

```diff
@@ -68,6 +68,13 @@ astropy.tests
 astropy.time
 ^^^^^^^^^^^^
 
+- Add new time format ``unix_ptp``, that monotonically
+  counts number of seconds (no leap seconds are introduced)
+  since ``1970-01-01T00:00:00 TAI``.
+  This format is similar but different to ``unix_tai`` which uses an epoch
+  of ``1970-01-01T00:00:08 TAI`` and ``unix`` which uses an epoch of
+  ``1970-01-01T00:00:00 UTC`` and which is subject to leap seconds. [#11143]
```

Comment:
```suggestion
- Add new time format ``unix_ptp`` (Precision Time Protocol) which provides 
  the number of SI seconds since ``1970-01-01T00:00:00 TAI``. [#11143]
```